### PR TITLE
Moving pushdown artifacts to the relevant profiles

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,148 +1,158 @@
...
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.google.cloud.spark</groupId>
-    <artifactId>spark-bigquery-parent</artifactId>
-    <version>0.24.0-SNAPSHOT</version>
-    <relativePath>spark-bigquery-parent</relativePath>
-  </parent>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (VinayLondhe14) [inline pom.xml:2]: Are the spacing changes intended?

File: pom.xml [hunk 2]
```diff
@@ -1,148 +1,158 @@
...
-    <module>spark-bigquery-connector-common</module>
-    <module>spark-bigquery-python-lib</module>
-    <module>spark-bigquery-pushdown</module>
-  </modules>
+    <modules>
+        <module>spark-bigquery-parent</module>
+        <module>bigquery-connector-common</module>
+        <module>spark-bigquery-tests</module>
+        <module>spark-bigquery-connector-common</module>
+        <module>spark-bigquery-python-lib</module>
```
Comment (VinayLondhe14) [inline pom.xml:53]: Do we not need `spark-bigquery-pushdown` here?

Comment (davidrabinowitz) [reply pom.xml]: No, it is not used in v2 at the moment, and there's no point to compile both 2.11 and 2.12 every time

**Review by VinayLondhe14 — APPROVED** [review]

**Pull request merged**